somefile.py

```diff
@@ -42,3 +42,8 @@ def run_command(
 def do_thing():
     print 'thinger'
     print 'thing 2'
+
+def things():
+    print 'thinger'
+    from urlparse import urlparse
```

Comment:
W1618 import missing `from __future__ import absolute_import`
W1648 Module moved in Python 3